migrate.md

```diff
@@ -0,0 +1,111 @@
+---
+
+copyright:
+  years: 2015, 2019
```

Comment:
Embarrassingly pedantic. but you probably want to use `years: 2019` since it's a new page.

Reply:
i always thought the copyright dates applied to how long the product itself has been in use. 
from http://rtpdoc01.rtp.raleigh.ibm.com:9082/kc/idibm/com.ibm.utic.publishing.doc/doc/notices_books.html#HDREDNOTICES: "Replace yyyy in the legal text shown to reflect your product copyright statement. Exclude the second year for first editions."

Reply:
That sounds fine.  Thanks for checking.